projects/packages/masterbar/src/admin-menu/class-admin-menu.php

```diff
@@ -117,14 +116,6 @@ public function add_my_mailboxes_menu() {
 		add_menu_page( __( 'My Mailboxes', 'jetpack-masterbar' ), __( 'My Mailboxes', 'jetpack-masterbar' ), 'manage_options', 'https://wordpress.com/mailboxes/' . $this->domain, null, 'dashicons-email', 4.64424 );
 	}
 
-	/**
-	 * Adds Stats menu.
-	 */
-	public function add_stats_menu() {
-		// @phan-suppress-next-line PhanTypeMismatchArgumentProbablyReal -- Core should ideally document null for no-callback arg. https://core.trac.wordpress.org/ticket/52539.
-		add_menu_page( __( 'Stats', 'jetpack-masterbar' ), __( 'Stats', 'jetpack-masterbar' ), 'view_stats', 'https://wordpress.com/stats/day/' . $this->domain, null, 'dashicons-chart-bar', 3 );
-	}
```

Comment:
I think we need to keep this in the `Jetpack_Admin_Menu` class in the time being.